components/work/WorkCreate.tsx

```diff
@@ -0,0 +1,34 @@
+import { FormEvent, PureComponent } from 'react';
+import { Container } from 'react-bootstrap';
+import { formToJSON } from 'web-utility';
+
+import { WorkEditor } from './WorkEditor';
+import activityStore from '../../models/Activity';
+import { TeamWorkModel } from '../../models/Team';
+import { NextRouter, withRouter } from 'next/router';
+class WorkCreate extends PureComponent<{
+  router: NextRouter | any;
+}> {
+  store = activityStore
+    .teamOf(this.props.router.query.name)
+    .workOf(this.props.router.query.tid);
+  handleSubmit = async (event: FormEvent<HTMLFormElement>) => {
+    event.preventDefault();
+    event.stopPropagation();
+    const data = formToJSON(event.target as HTMLFormElement);
+    await this.store.updateOne(data);
```

Comment:
```suggestion
    await this.store.updateOne(data, this.props.wid);
```
可根据路由是否匹配到作品 ID 自动控制是新增还是修改，让作品新增、编辑页复用一份代码。